graphql_compiler/cost_estimation/statistics.py

```diff
@@ -178,3 +225,25 @@ def get_field_quantiles(self, vertex_name, field_name):
         """See base class."""
         statistic_key = (vertex_name, field_name)
         return self._field_quantiles.get(statistic_key)
+
+    def get_value_count(self, vertex_name: str, field_name: str, value: Any) -> Optional[int]:
+        """See base class."""
+        vertex_sampling_summary = self._sampling_summaries.get(vertex_name)
+        if vertex_sampling_summary is None:
+            return None
+
+        field_sampled_value_counts = vertex_sampling_summary.value_counts.get(field_name)
+        if field_sampled_value_counts is None:
+            return None
+
+        sampled_value_count = field_sampled_value_counts.get(value)
+        if sampled_value_count:
```

Comment:
Let's be explicit here. Is this a `is not None` or a `> 0` or both or something else entirely?